client/app/account/billing/main/translations/Messages_fr_FR.xml

```diff
@@ -4,4 +4,17 @@
    <translation id="billing_main_pay_as_you_go_tab" qtlid="497608">Pay As You Go</translation>
    <translation id="billing_main_bills_tab" qtlid="20179">Factures</translation>
    <translation id="billing_main_payments_tab" qtlid="398781">Suivi des paiements</translation>
+   <translation id="billing_main_bill_download_error">La facture suivante n’a pas pu être téléchargée :</translation>
+   <translation id="billing_main_bill_download_all_error">Une erreur est survenue lors du téléchargement des factures</translation>
+   <translation id="billing_main_bill_download_action">Télécharger plusieurs factures</translation>
+   <translation id="billing_main_bill_download_title">Téléchargement des factures</translation>
+   <translation id="billing_main_bill_archive_name">Nom de l’archive</translation>
+   <translation id="billing_main_bill_download_date_range">Période</translation>
+   <translation id="billing_main_bill_download_date_from" qtlid="88725">Du</translation>
+   <translation id="billing_main_bill_download_date_to" qtlid="226227">Au</translation>
+   <translation id="billing_main_bill_download_confirm" qtlid="39286">Télécharger</translation>
+   <translation id="billing_main_bill_download_date_range_invalid">Veuillez sélectionner une période inférieure ou égale à 1 mois</translation>
```

Comment:
You can't download bills for more than a month ?

Reply:
For 504 purpose, we limit for the moment to only one month

Reply:
Couldn't APIv7 or batchMode solve this issue ?